microservice/challenge-1/ReminderAppl/README.md

```diff
@@ -0,0 +1,32 @@
+Context:
```

Comment:
Missing markdown formatting.
Missing steps to build locally, step to run locally, steps to run with docker and docker-compose.
Steps to package and deploy helm chart.